bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -59,6 +60,7 @@ type InitConfiguration struct {
 	// The list of phases can be obtained with the "kubeadm init --help" command.
 	// This option takes effect only on Kubernetes >=1.22.0.
 	// +optional
+	// +kubebuilder:validation:MaxItems=100
 	SkipPhases []string `json:"skipPhases,omitempty"`
```

Comment:
How many phases does KubeADM have? 😱

Reply:
I would guess 15-20 😃. We can definitely talk to down a bit. @fabriziopandini wdyt?

Reply:
I will use 50

Reply:
Done